teams/vsp/teams/tools/frontend/support-and-triage.md

```diff
@@ -0,0 +1,69 @@
+# Support and Triage rotation 
+
+## Overview 
+
+Supporting VSA teams requires a cohesive and well-coordinated experience. The frontend tools team assigns responsibility for this work to a VSP front end engineer on a weekly rotation. This document summarizes the responsibilities of this role and how the rotation works. 
```

Comment:
how do we refer to this person? the "support dev"? might be useful to have a role name (to differentiate from "oncall" etc.)

Reply:
support dev works.